.gitignore

```
@@ -0,0 +1,2 @@
+text.md
```

Kindly fix the misspelling of the `text.md` file as it should be renamed `test.md`, both locally and remotely.
```suggestion
test.md
```